manifests/port.pp

```diff
@@ -36,20 +36,17 @@
 #  }
 #
 define selinux::port (
-  $context  = undef,
+  $context,
+  $port,
   $protocol = undef,
-  $port     = undef,
-  $policy   = 'targeted'
 ) {
-  Exec {
-    path => '/bin:/sbin:/usr/bin:/usr/sbin',
-  }
 
-  if ( $context == undef ) or ( $port == undef ) {
```

Comment:
I would prefer this stays. That way, the module controls the error message rather than Puppet throwing an error with less than useful info.